plugins/services/src/js/components/ServiceStatusIcon.tsx

```diff
@@ -104,35 +123,33 @@ class ServiceStatusIcon extends Component {
     );
   }
 
-  getUnableToLaunchWarning(service) {
-    if (this.isUnableToLaunch(service)) {
-      return this.getTooltip(
-        <Trans render="span">
-          There are tasks in this queue that DC/OS has failed to deploy for{" "}
-          {DateUtil.getDuration(
-            Date.now() - DateUtil.strToMs(service.getQueue().since)
-          )}
-          .
-        </Trans>
-      );
-    }
+  getUnableToLaunchWarning(service: Service) {
```

Comment:
This can also be a pod, right?

Reply:
A `Pod` is a `Service`. But you're right in that we most likely can get away with declaring `ServiceOrPodOrTree` like so:

```
type ServiceTreeNode = 
  Service | ServiceTree
```

Thanks for the nudge!